cmd/gateway-main.go

```diff
@@ -136,14 +136,9 @@ FLAGS:
   {{end}}{{end}}
 
 ENVIRONMENT VARIABLES: (Default values in parenthesis)
-  SIA_MANAGER_DELAY_SEC:     The number of seconds to delay between cache/db management operations. (30)
-  SIA_UPLOAD_CHECK_FREQ_MS:  The time in milliseconds to wait between checks with Sia network to determine if file has completed uploading. (3000)
-  SIA_CACHE_MAX_SIZE_BYTES:  The maximum allowed size of the cache directory in bytes. (10000000000)
-  SIA_CACHE_PURGE_AFTER_SEC: The number of seconds since the time a file was last downloaded before removing that file from the cache. (86400)
```

Comment:
how long will the cached files purged then?

Reply:
All cache functionality has been removed.